experimental/logging/log_listener.go

```diff
@@ -96,9 +109,9 @@ func (l *loggingListener) writeIndented(before bool, err error, vals []uint64, i
 		l.writeFuncEnter(&message, vals)
 	} else { // after
 		if l.fnd.GoFunction() != nil {
-			message.WriteString("<== ")
+			message.WriteString("<==")
 		} else {
-			message.WriteString("<-- ")
+			message.WriteString("<--")
```

Comment:
this stops writing `<-- ()` for no results in favor of `<--`

Reply:
sg